FORK.md

```diff
@@ -1,7 +1,26 @@
 # Fork
 
-## optimism
+## Optimism
 
 - tag: `v1.0.3`
 - differences
   - [Differences from Optimism Bedrock](./specs/differences-from-optimism-bedrock.md)
+
+### Others
```

Comment:
I think this `FORK.md` should only include the difference from Optimism.
Why don't we remove these list in `FORK.md`, and leave in this PR's description?

Reply:
I added this list to match with [this change](https://github.com/kroma-network/kroma/pull/136/files#diff-93ec59db0ab516baf2c0eebb87cf68bbac382c2d5bff18c1aa8f9abca3ee729d). I agree this list should be removed after applying upstream changes by tag like `v1.0.7`. Wdyt?

Reply:
Ok. Let's remove those list after applying upstream changes.